packages/cli/src/databases/migrations/sqlite/1724951148974-AddApiKeysTable.ts

```diff
@@ -0,0 +1,74 @@
+import type { ApiKey } from '@/databases/entities/api-key';
+import type { MigrationContext } from '@/databases/types';
+import { generateNanoId } from '@/databases/utils/generators';
+
+export class AddApiKeysTable1724951148974 {
+	async up({ queryRunner, tablePrefix, runQuery }: MigrationContext) {
+		const tableName = `${tablePrefix}user_api_keys`;
+
+		// Create the table
+		await queryRunner.query(`
+			CREATE TABLE ${tableName} (
+				id VARCHAR(36) PRIMARY KEY NOT NULL,
+				"userId" VARCHAR NOT NULL,
+				"label" VARCHAR(255) NOT NULL,
```

Comment:
The other table is `varchar(100)`. Let's make sure everything matches between different databases